### ISSUE TITLE
Using an expression with value_for_material with material selection crashes VisIt.

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#589]
Kevin Roe showed me where adding a plot with an expression that used value_for_material crashed VisIt. I haven't been able to reproduce it with rect2d.silo or ucd3d.silo, but it immediately crashed for Kevin, perhaps it has to do with running in parallel.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 05/24/2017 04:36 pm
Original update: 05/26/2017 01:22 pm
Ticket number: 2831
